lib/rules/no-duplicate-at-import-rules/README.md

```diff
@@ -0,0 +1,49 @@
+# no-duplicate-at-import-rules
+
+Disallow duplicate `@import` rules within a stylesheet.
+
+```css
+    @import "a.css";
+    @import "a.css";
+/** ↑
+ * These are duplicates */
+```
+
```

Comment:
Sorry for being so late on the ball, but could we get a line here saying something like "This rule ignores `@import` in Less.". 

This is because `postcss-less` doesn't omit `AtRule` nodes, but `Import` nodes and we don't support that anyway.